.github/workflows/ci.yml

```
@@ -0,0 +1,103 @@
+  push:
+    branches: [ "main", "develop" ]
+  pull_request:
+
+jobs:
+  detect-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      payment: ${{ steps.filter.outputs.payment }}
+      platform: ${{ steps.filter.outputs.platform }}
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)